lib/apiservers/engine/backends/system.go

```diff
@@ -93,6 +94,10 @@ func (s *System) SystemInfo() (*types.Info, error) {
 		log.Infof("System.SytemInfo unable to get global status on containers: %s", err.Error())
 	}
 
+	ctx, cancel := context.WithTimeout(context.Background(), infoTimeout)
```

Comment:
Can we also note in the `docker info` output that we're using Admiral and a name/address if we have one available at this point.